src/basicpy/basicpy.py

```diff
@@ -415,11 +403,11 @@ def fit(
                 B,
                 I_R,
             )
-            logger.info(f"single-step optimization score: {norm_ratio}.")
-            logger.info(f"mean of S: {float(jnp.mean(S))}.")
+            logger.debug(f"single-step optimization score: {norm_ratio}.")
+            logger.debug(f"mean of S: {float(jnp.mean(S))}.")
             self._score = norm_ratio
             if not converged:
-                logger.warning("single-step optimization did not converge.")
+                logger.debug("single-step optimization did not converge.")
             if S.max() == 0:
                 logger.error("S is zero. Please try to decrease smoothness_darkfield.")
```

Comment:
I changed this to debug because every iteration it does not converge we get a message. This does not appear useful as a warning, since it is probably common and not significant that it does not converge on every iteration. What is more significant is that it does not converge at all, and that is contained in a warning below.

I also made this debug instead of info because I tried to limit info to messages that would give a progress update. I figure debug is important to see the details of each step, but info is only important to see progress of each step. I have one info message for each loop.

If you still think this should be changed to info, I will change it.